packages/conf-rdkit/conf-rdkit.1/opam

```diff
@@ -6,7 +6,7 @@ bug-reports: "https://github.com/ocaml/opam-repository/issues"
 license: "BSD-3-Clause"
 build: [
   ["sh" "-c" "c++ test.cpp -o test -I/usr/local/include/rdkit -L/usr/local/lib -lRDKitRDGeneral -DSWIG"] {os = "macos"}
-  ["sh" "-c" "c++ test.cpp -o test -I/usr/include/rdkit -lRDKitRDGeneral -DSWIG || c++ test.cpp -o test -I/usr/include/rdkit -lRDGeneral -DSWIG"] {os = "linux"}
+  ["sh" "-c" "c++ test.cpp -o test -I/usr/include/rdkit -lRDKitRDGeneral -DSWIG || c++ test.cpp -o test -I/usr/include/rdkit -lRDGeneral -DSWIG"] {os = "linux" | os = "freebsd"}
```

Comment:
it's more likely you'll need to go the macos path... headers in /usr/local/include/rdkit; libraries in /usr/local/lib:

`-I/usr/local/include/rdkit -L/usr/local/lib`

Reply:
Thanks Hannes! Second attempt in a17dc33...